src/compiler/scala/tools/nsc/backend/ScalaPrimitivesOps.scala

```diff
@@ -0,0 +1,238 @@
+/* NSC -- new Scala compiler
+ * Copyright 2005-2013 LAMP/EPFL
+ * @author  Martin Odersky
+ */
+
+package scala
+package tools.nsc
+package backend
+
+object ScalaPrimitivesOps extends ScalaPrimitivesOps
```

Comment:
There is still an `object ScalaPrimitives` for compatibility: https://github.com/DarkDimius/scala/pull/7/files#diff-859cee54c124f77e4b12fe371a1198a6R411, what else is needed?